package.json

```diff
@@ -214,8 +214,6 @@
     "prettier": "1.19.1",
     "progress-bar-webpack-plugin": "^2.1.0",
     "puppeteer": "^2.1.1",
-    "react": "^16.8.4",
```

Comment:
any reason why this is removed?

Reply:
Hello,
Actually I removed this because I thought having react as a dependency there was the cause of an error when trying to `npm link` the Kepler package (@see https://stackoverflow.com/questions/66975613/error-invalid-hook-call-hooks-can-only-be-called-inside-of-the-body-of-a-funct), but it appeared it wasn't the reason as we could not get rid of this error.

As we could not `npm link` we resolved to pushing the `dist/` folder on our repo and updating our dependency to point to our repo, hence the presence of the `dist/` folder.

Concerning this PR, we were not sure it could be accepted as we made the assumption that an array typed column in the CSV would be in a JSON format. This is the format we decided to use for our project, but it may not be the case for everyone, and maybe it should be discussed.

If the changes can be accepted, I'll remove the `dist/` from this branch as well as the dependency change.